app/views/govuk_publishing_components/components/related_navigation/_section.html.erb

```diff
@@ -3,27 +3,36 @@
      aria-labelledby="related-nav-<%= section_title %>-<%= random %>"
      data-module="gem-toggle">
 
-  <% unless section_title === "related_items" %>
+  <% if section_title === "related_items" %>
+   <% heading_text = related_nav_helper.component_title %>
+  <% else %>
     <%=
       heading_class = related_nav_helper.section_css_class("gem-c-related-navigation__sub-heading", section_title)
       heading_data = { 'track-count' => related_nav_helper.section_data_track_count(:related_item_section) }
-      heading_id = "related-nav-#{section_title}-#{random}"
+      heading_label = "related-nav-#{section_title}-#{random}"
       heading_level = related_nav_helper.section_heading_level
+      heading_text = related_nav_helper.construct_section_heading(section_title)
 
-      content_tag(heading_level, id: heading_id, class: heading_class, data: heading_data) do
-        related_nav_helper.construct_section_heading(section_title)
+      content_tag(heading_level, class: heading_class, data: heading_data, aria: { label: heading_label }) do
+        heading_text
```

Comment:
This should be `aria-labelledby` otherwise the assistive technology will read out the heading literally as `related-nav-related_items-dde33a2c`

Reply:
Actually it should be id, which is what it was in the first place. I think I went in a bit of a circle trying to fix an accessibility warning in the component guide, but I've concluded that the markup was fine as it was, I can't account for the warning. Have switched it back to id instead.

Reply:
@andysellick Good spot, thanks